assets/app/scripts/app.js

```diff
@@ -286,7 +286,7 @@ angular
       });
   })
   .constant("API_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).api))
-  .constant("APIS_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).apis))
+  .constant("APIS_CFG", (window.OPENSHIFT_CONFIG || {}).apis)
```

Comment:
yeah i have no problem with the different constants pointing to different
pieces, but cloning causes problems with setting the content dynamically

On Thu, Apr 7, 2016 at 2:01 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In assets/app/scripts/app.js
> <https://github.com/jwforres/origin/pull/38#discussion_r58916901>:
>
> > @@ -286,7 +286,7 @@ angular
> >        });
> >    })
> >    .constant("API_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).api))
> > -  .constant("APIS_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).apis))
> > +  .constant("APIS_CFG", (window.OPENSHIFT_CONFIG || {}).apis)
>
> I think it was to scope down which bits of the config were needed by
> different parts
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jwforres/origin/pull/38/files/bd96d77d46bc3164e90368385a2d34e2f37213fd#r58916901>
>